OxfordBodleian/Juel-Jensen/BDLaethe38.xml

```diff
@@ -136,51 +412,171 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                         <dim type="left">11</dim>
                                         <dim type="intercolumn">9</dim>
                                     </dimensions>
+                                    <note> The characters per line are 6-9.</note>
+                                    <ab type="pricking">Ruling and pricking are visible. Both primary and ruling pricks are visible.</ab>
+                                    <ab type="ruling" subtype="pattern">Ruling pattern: 1A-1A-1A1A/0-0/0-0/C.</ab>
+                                    <ab type="ruling"> The upper line is written above the ruling. The bottom line is written above the ruling.</ab>
                                 </layout>
                             </layoutDesc>
-                            
                         </objectDesc>
                         
                         <handDesc>
                             <handNote xml:id="h1" script="Ethiopic">
-                           
+                                <seg type="script"/>
+                                <persName ref="PRS12986GabraKr"/>
+                                <seg type="ink">Black, red</seg>
+                                <date notBefore="1775" notAfter="1815"></date>
+                                <desc>Fine and regular handwriting, slightly deteriorating towards the end (e.g. <locus from="76v" to="78v"/>).
+                                </desc>
+                                <seg type="rubrication">
+                                    Holy names and the names of the individuals involved in the production of the book, both in the colophon and in the supplication formulas throughout the
+                                    manuscript. Several groups of lines on the incipit page of <ref target="#i2"/> and of the opening sections for each weekday, sometimes alternating with black lines. 
```

Comment:
the value of target should be "ms_i1" etc. If you type # in oxygen, the schema should prompt the list of all xml:ids of the file, from which you can chose the one you need (to avoid typos)